website/content/docs/configuration/storage/yugabytedb-ysql.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: YugabyteDB (YSQL)  - Storage Backends - Configuration
+description: |-
+  The YugabyteDB (YSQL)   storage backend is used to persist Vault's data in a YugabyteDB (YSQL)  
+  server or cluster.
+---
+
+# YugabyteDB (YSQL)   Storage Backend
+
+The YugabyteDB (YSQL)   storage backend is used to persist Vault's data in a
+[YugabyteDB(YSQL)](https://www.yugabyte.com)
+
+```hcl
+storage "yugabytedb_ysql" {
+  connection_url = "postgres://user123:secret123!@localhost:5432/vault"
+}
+```
+
+~> **Note:** The YugabyteDB (YSQL)   storage backend plugin will attempt to use SSL
+when connecting to the database. If SSL is not enabled the `connection_url`
+will need to be configured to disable SSL. See the documentation below
+to disable SSL.
+
+The YugabyteDB (YSQL)   storage backend does not automatically create the table. Here is
```

Comment:
Plugins available for Postgres follow the same mechanism.
Though the method of creating the tables automatically can also be used.
My thoughts:
1. It may lead to providing the user with create table permissions to the hashicorp vault.
2. If two vault instances are connected to the same database for storage it may be problematic. (In our case we can have tables with a different name.)